patches/0076-qsv-use-a-new-method-to-create-mfx-session-when-usin.patch

```diff
@@ -1007,6 +1013,44 @@ index a8bc8c8143..d3c061b739 100644
      return ret;
  }
  
+@@ -1523,6 +1746,13 @@ static int qsv_device_create(AVHWDeviceContext *ctx, const char *device,
+         }
+     } else if (CONFIG_VAAPI) {
+         child_device_type = AV_HWDEVICE_TYPE_VAAPI;
++#if QSV_ONEVPL
++    } else if (CONFIG_D3D11VA) {  // Use D3D11 by default if d3d11va is enabled
++
```

Comment:
av_log(NULL, AV_LOG_WARNING,
                "WARNING: defaulting child_device_type to CONFIG_D3D11VA for oneVPL"
                "Please explicitly set device type via \"-init_hw_device\" option. if needed\n");

Reply:
Thanks, I updated the PR.